tests/unit/sdk/checks/conftest.py

```diff
@@ -17,5 +17,6 @@ async def mock_gql_query_my_query(httpx_mock: HTTPXMock) -> HTTPXMock:
         method="POST",
         json=response,
         url="http://localhost:8000/api/query/my_query?branch=main&update_group=false",
+        is_reusable=True,
```

Comment:
How is that related to this change ?

Reply:
I created a dedicated PR to upgrade `pytest-httpx` #354

Reply:
Yes it was actually related to pytest-httpx... I've rebased this branch on top of yours